plugins/modules/capabilities.py

```diff
@@ -61,6 +61,12 @@
     path: /bar
     capability: cap_net_bind_service
     state: absent
+
+ - name: Remove all capabilities from /restricted_app
+   community.general.capabilities:
+    path: /restricted_app
+    capability: ""
```

Comment:
Are you sure that passing an empty string here (with `state: absent`) actually removes all capabilities? I tried to follow the code and cannot see how that happens.

When `capability == ""`, `self.capability_tup` should end up as `("", None, None)`. Since `""` is not likely in `caps`, the module will simply exit with `changed=False`.

Do I miss something?